tests/bundle/Functional/SessionTest.php

```diff
@@ -134,17 +134,38 @@ public function testLoginWithExistingFrontendSession()
         $response = $this->sendHttpRequest($request);
 
         // Session is recreated when using CSRF, expect 201 instead of 200
-        self::assertHttpResponseCodeEquals($response, 201);
+        $this->assertHttpResponseCodeEquals($response, 201);
     }
 
-    /**
-     * @depends testDeleteSession
-     */
-    public function testDeleteSessionExpired($session)
+    public function testDeleteSessionExpired(): void
     {
-        $response = $this->sendHttpRequest($this->createDeleteRequest($session));
-        self::assertHttpResponseCodeEquals($response, 404);
+        $session = $this->login();
+        $deleteSessionRequest = $this->createDeleteRequest($session);
+
+        $response = $this->sendHttpRequest($deleteSessionRequest);
+
+        $this->assertHttpResponseCodeEquals($response, 204);
         self::assertHttpResponseDeletesSessionCookie($session, $response);
+
+        //triggered again to make sure deleting already deleted session results in 404
```

Comment:
It's a super small nitpick but my OCD kicks in ;)

```suggestion
        // Triggered again to make sure deleting already deleted session results in 404
```